modules/apps/layout/layout-reports-web/src/main/resources/META-INF/resources/js/components/IssueDetail.js

```diff
@@ -60,3 +69,137 @@ HtmlPanel.propTypes = {
 	content: PropTypes.string.isRequired,
 	title: PropTypes.string.isRequired,
 };
+
+const FailingElementsPanel = ({failingElements, issueType}) => {
+	const [shownElements, setShownElements] = useState(10);
+
+	const normalizedFailingElements = normalizeFailingElements(
```

Comment:
Not sure how many elements we may receive here, but if it can be many we can add an useMemo here to avoid iterating over the list in each render

Reply:
I thought about it, but the IssueDetail component is rendered only once. Its content is static and there's nothing in it that make it render again. If that's the case, a useMemo wouldn't do anything, right?

Reply:
Clarified in Slack!